README.md

```diff
@@ -409,26 +409,25 @@ If you plan to work with data from all missions, then the download will require
 
 ### Full ISIS Data Download
 
-The ISIS Data Area is hosted on rsync servers and not through conda channels like the ISIS binaries. This requires using the rsync command from within a terminal window within your Unix distribution, or from within WSL if running Windows 10.  Downloading all mission data requires over 520 GB of disk space. If you want to acquire only certain mission data [click here](#Mission-Specific-Data-Downloads). To download all ISIS data files, continue reading.
+> Warning if you are looking to download ISIS data via rsync the servers will be shutdown in a few months.
+the outdated rsync download information can be found [here](https://github.com/USGS-Astrogeology/ISIS3/wiki/Outdated-ISIS-Data-Information)
 
-To download all ISIS data, enter the following commands in the location where you want to install the ISIS Data Area, for versions of ISIS 4.1.0 and later:
 
-    cd $ISISDATA
-    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isisdata/data/ .
+The ISIS Data Area is hosted on a combination of AWS S3 buckets and public http servers e.g. NAIF, Jaxa, ESA and not through conda channels like the ISIS binaries. This requires using the `downloadIsisData.py` script from within a terminal window within your Unix distribution, or from within WSL if running Windows 10. Downloading all mission data requires over 520 GB of disk space. If you want to acquire only certain mission data [click here](#Mission-Specific-Data-Downloads). To download all ISIS data files, continue reading.
+
+To download all ISIS data, use the following command:
 
-For earlier versions, use:
+    downloadIsisData sync all $ISISDATA
 
-    cd $ISIS3DATA
-    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/ .
+> Note: this applicaion takes in 3 parameters in the following order \<rclone command> \<mission> \<download destination>
 
 > Note: The above command downloads all ISIS data including the required base data area and all of the optional mission data areas.
 
 ### Partial Download of ISIS Base Data
 
 This data area contains data that is common between multiple missions such as DEMS and leap second kernels. As of ISIS 4.1, the base data area is no longer required to run many applications as data such as icons and templates has been moved into the binary distribution. If you plan to work with any applications that use camera models (e.g., cam2map, campt, qview), it is still recommended you download the base data area. To download the base data area run the following commands:
 
-    cd $ISISDATA
-    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isisdata/data/base .
+    downloadIsisData sync base $ISISDATA
 
 For versions of ISIS prior to ISIS 4.1.0, please use `isis3data` instead of `isisdata` in the above command.
```

Comment:
This line is no longer correct. How do you download the ISIS 3 data area now?

Reply:
@jessemapel We still need to make a change to the downloading script to support this. Once we have that there will be an optional flag to download the ISIS 3 data area. Once we have that I'll update the readme to include how to set the flag.